codeception.yml

```diff
@@ -4,6 +4,6 @@ include:
     - frontend
     - backend
 paths:
-    log: console/runtime/logs
+    output: console/runtime/output
```

Comment:
is this needed, log seems just fine - and could potentially break some existing stuff?

Reply:
log is dropped in favour of output in codeception 2.3 so there should not be any breaking changes (all my tests passed locally) 

See here https://github.com/Codeception/Codeception/issues/4688#issuecomment-354017094

My understanding was that log was transposed to output in the background for some time (for backward compatibilty before being dropped altogether which causes it to now error)

Original issue -> https://github.com/Codeception/Codeception/issues/4688